docs/topics/Gateway.md

```diff
@@ -684,6 +688,47 @@ Sent when a guild role is deleted.
 | guild_id | snowflake | id of the guild |
 | role_id  | snowflake | id of the role  |
 
+### Invites
+
+### Invite Create
+
+Send when a new invite to a channel is created.
+
+###### Invite Create Event Fields
+
+| Field      | Type                                                                          | Description                                                                                                        |
+| ---------- | ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| channel_id | snowflake                                                                     | the channel the invite is for                                                                                      |
+| code       | string                                                                        | the unique invite [code](#DOCS_RESOURCES_INVITE/invite-object)                                                     |
+| created_at | timestamp                                                                     | the time at which the invite was created                                                                           |
+| guild_id   | snowflake                                                                     | the guild of the invite                                                                                            |
+| inviter    | [partial user object](#DOCS_TOPICS_GATEWAY/invite-create-partial-user-object) | the user that created the invite                                                                                   |
+| max_age    | int                                                                           | how long the invite is valid for (in seconds)                                                                      |
+| max_uses   | int                                                                           | the maximum number of times the invite can be used                                                                 |
+| temporary  | boolean                                                                       | whether or not the invite is temporary (invited users will be kicked on disconnect unless they're assigned a role) |
+| uses       | int                                                                           | how many times the invite has been used (always will be 0)                                                         |
+
+###### Invite Create Partial User Object
```

Comment:
It seems that the event also provides the nullable `User#is_bot` field. Is it intended ? And if yes, why is it described as a partiel user object ?